lib/validator/formchecker.cpp

```diff
@@ -1371,63 +1378,63 @@ Expect<void> FormChecker::checkInstr(const AST::Instruction &Instr) {
   // Memory Instructions.
   case OpCode::I32__load:
     return checkAlignAndTrans(32, {ValType(TypeCode::I32)},
-                              {ValType(TypeCode::I32)});
+                              {ValType(TypeCode::I32)}, false);
```

Comment:
As above.
The lots of changes bellow are not necessary with the above description.

Reply:
I see, thanks